.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+tmp
```

Comment:
You're right. I think I've found a better way to support the `mocha --bail` workflow.